markdown.ipynb

```diff
@@ -4,21 +4,22 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `markdown.ipynb` Cell 1 Line 3_
```markdown
> ## This is another header.
> 
> 1.   This is the `first` list **item**.
```
Review comment

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/24)._